docassemble/AssemblyLine/data/questions/al_reminders.yml

```diff
@@ -87,12 +102,23 @@ code: |
   # In addition, this cron will stop ALL cron tasks on the session once it runs through to the end. If you have other
   # crons that still need to run, you need to replace this block as well.
 
+  reconsider("al_reminders_cron_daily")
+
+  # Evaluate whether all of the tasks in al_reminders_cron_daily are finished. If so, stop running
+  # cron so that last modified date of session won't keep refreshing and sessions can be automatically
+  # cleaned up
+  reconsider("al_reminders_evaluate_stop_cron")
+
+  # Every cron event should end with response()
+  response()
+---
+code: |
   if al_user_wants_reminders:
     import time
     from random import random
     # Avoid all of the interviews using up resources at the same time, vary the start time a bit
     time.sleep(random()*10)
-
+    log("Running cron")
```

Comment:
Is this log here for development debugging, or is it meant to stay here? If it's meant to stay, could it be more specific to AL?

Reply:
Just a temporary patch, it's not permanent. No need to review this commit-I had to save work while switching from work to home PC.